Tests/NIOPosixTests/EventLoopTest.swift

```diff
@@ -184,6 +184,26 @@ public final class EventLoopTest : XCTestCase {
         XCTAssertEqual(error as? EventLoopError, .cancelled)
     }
 
+    public func testScheduledTasksAreOrdered() throws {
+        let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
```

Comment:
Can we have a version of this test for `EmbeddedEventLoop` too?

Reply:
There is a test for the `EmbeddedEventLoop` already that tests the ordering `testScheduledTasksInOrder`. It looks slightly different but has the same effect. Is that one enough or do you want me to add the exact same test as well?

Reply:
That should be fine.